src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,48 @@
+"""Kubectl Client BCI container"""
```

Comment:
NIT: maybe "Kubernetes Client" or simply "Kubectl" (without client). WDYT?

Reply:
I think we can use kubectl.